examples/README.md

```diff
@@ -0,0 +1,85 @@
+# krun example
+
+This directory contains a simple experiment using krun.
+This is a good starting point for setting up your own krun configuration.
+
+The example here contains two benchmark programs (*nbody* and *dummy*),
+executed on two VMs (*cPython* and a standard *JVM* such as HotSpot).
+Each benchmark is run for 5 iterations on the same VM, then the VM is
+restarted and the benchmark is re-run for another 5 iterations.
+We say that the benchmarks are *executed* twice, with 5 *iterations*.
+
+This configuration can be found in the file `examples/example.krun`.
+
+## Step 0: prepare the benchmarking machine
+
+To run this example experiment, you need `sudo` rights to the machine you are
+using, and you should have installed:
+
+  * Python
+  * a Java SDK (version 7)
+  * GNU make, a C compiler and libc (e.g. `sudo apt-get install build-essential`)
+  * cpufrequtils (e.g. `sudo apt-get install cpufrequtils`)
+  * cffi (e.g. `sudo apt-get install python-cffi`)
+
+You also need to set up some kernel arguments.
+To do this on a Debian-based system (including Ubuntu):
+
+  * Edit /etc/default/grub (e.g. `sudo gedit /etc/default/grub`)
+  * Add `isolcpus=X` to `GRUB_CMDLINE_LINUX_DEFAULT` (where `X` is an integer > 0)
+  * Run `sudo update-grub`
+
+Create a new user called `krun`, with minimal permissions:
+
+```bash
+sudo useradd krun
+```
+
+## Step 1: Fetch the krun source
+
+```bash
+$ git clone https://github.com/softdevteam/krun.git
+$ cd krun
+```
+
+## Step 2: set `JAVA_HOME`
+
+```bash
+$ export JAVA_HOME=/usr/lib/jvm/java-7-openjdk-amd64/
+```
+## Step 3: Build krun
+
+```bash
+$ pwd
+.../krun
+$ make JAVA_CPPFLAGS='"-I${JAVA_HOME}/include -I${JAVA_HOME}/include/linux"' \
+    JAVA_LDFLAGS=-L${JAVA_HOME}/lib ENABLE_JAVA=1
```

Comment:
You can also set the `CC` make variable to choose a C compiler. Worth mentioning?